bazel/external/lightstep.BUILD

```diff
@@ -0,0 +1,38 @@
+load("@//bazel/external:helpers.bzl", "cc_genrule_deps")
+
+cc_library(
+    name = "lightstep",
+    srcs = [":_prefix/lib/liblightstep_core_cxx11.a"],
+    hdrs = glob([
+        "src/c++11/lightstep/**/*.h",
+        "src/c++11/mapbox_variant/**/*.hpp",
+    ]) + [
+        ":_prefix/include/collector.pb.h",
+        ":_prefix/include/lightstep_carrier.pb.h",
+    ],
+    includes = ["src/c++11", "_prefix/include"],
+    deps = ["@protobuf_bzl//:protobuf"],
+    visibility = ["//visibility:public"],
+)
+
+cc_genrule_deps(
```

Comment:
Please add a comment on why this is needed, i.e. that you need the protobuf library headers/libraries to feed into `GENRULE_REPOSITORY_COMMAND` below.

Reply:
Done.